package/libs/openssl/Makefile

```diff
@@ -184,7 +190,7 @@ define Build/Configure
 			--openssldir=/etc/ssl \
 			$(TARGET_CPPFLAGS) \
 			$(TARGET_LDFLAGS) -ldl \
-			-DOPENSSL_SMALL_FOOTPRINT \
+			$(if $(value CONFIG_OPENSSL_OPTIMIZE_SPEED),,-DOPENSSL_SMALL_FOOTPRINT) \
```

Comment:
Okay, I'll bite... Why aren't we using:

`$(if $(CONFIG_OPENSSL_OPTIMIZE_SPEED),,-DOPEN_SSL_SMALL_FOOTPRINT)`

I can't find a single example of using `$(if $(value ...))` in the entire tree.

Reply:
Well, it did work however I've switched to your example instead. Thanks!